sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_utils.py

```diff
@@ -58,8 +59,8 @@ def build_call_locator(
     server_call_id: Optional[str],
     group_call_id: Optional[str],
     room_id: Optional[str],
-    args: List[Union['ServerCallLocator', 'GroupCallLocator','RoomCallLocator']] = None,
-) -> CallLocator:
+    args: Optional[List[Union['ServerCallLocator', 'GroupCallLocator','RoomCallLocator']]] = None,
+) -> Optional[CallLocator]:
```

Comment:
the issue here, is that is None the right return here ?

request: Optional[CallLocator] = None

so a check if its None before turning will help you fix it, if instead of None an error is better. If None is right, then leave it as is

Reply:
Yeah I think `Optional` is okay here - everywhere this is used the result is None-checked.